libraries/AP_Scripting/docs/docs.lua

```diff
@@ -3011,6 +3011,12 @@ function terrain:status() end
 ---@return boolean
 function terrain:enabled() end
 
+--  Calculate lookahead rise in terrain. This returns extra altitude needed to clear upcoming terrain in meters
+--- @param bearing number -- direction to look ahead 
+--- @param distance number -- distance to look ahead
+--- @param climb_ratio number - assumed rate of climb of the vehicle
+--- @return number -- the hightest extra altitude reired to clear the terrain
+function terrain:lookahead(bearing, distance, climb_ratio) end
```

Comment:
You don't seem to be using this new method. You already have a implementation in lua? My vote would be to save flash and not have the binding.